### DISCUSSION TITLE
Unable to start -  so-soctopus seems broken

### DISCUSSION THREAD

**Discussion by mciantar** [Security-Onion-Solutions/securityonion#8362] [Unsupported Versions]
I cannot start my so instance after a reboot. 

running dockerd --debug shows the following:

DEBU[2022-07-22T11:35:01.710965417Z] form data: {"AttachStderr":true,"AttachStdin":false,"AttachStdout":true,"Cmd":["python3","playbook_play-sync.py"],"Detach":false,"DetachKeys":"","Env":null,"Privileged":false,"Tty":false,"User":"","WorkingDir":""}
ERRO[2022-07-22T11:35:01.711054162Z] Error setting up exec command in container so-soctopus: Container 1730d5b337e073bba8b345f263480f3fd86414a1cd43957b78e3c8ef19f4f208 is not running

Does anyone know if I can recover?

**Comment by weslambert** [comment]
When you say you cannot start your SO instance, are you referring to not being able to start any containers, or just `so-soctopus`?

What is the output of `so-status`?

What is the output of `docker logs so-soctopus`?

You might also try the following (as root):

`docker stop so-soctopus`
`docker rm so-soctopus`
`salt-call state.apply state.highstate queue=True`
